### PR TITLE
Feature/Fix-Permits-Not-Being-ETL'd

### PR DESCRIPTION
**Pull request by nayr974**

### CODE REVIEW

**Review by mmumby — APPROVED** [review]

**Review by pdittaro — APPROVED** [review]
Thanks for the fix

**Review by pdittaro — CHANGES_REQUESTED** [review]
Actually, this will do a weird side effect if we build out a new environment from scratch. I think its alright if we just get rid of that migration and then add the column as part of the stored procedure.

File: migrations/sql/V2020.09.03.14.10__etl_permit_security_adjustment.sql [hunk 1]
```diff
@@ -1,2 +0,0 @@
-ALTER TABLE ETL_PERMIT
-ADD COLUMN IF NOT EXISTS security_adjustment numeric;
```
Comment (Jsyro) [inline migrations/sql/V2020.09.03.14.10__etl_permit_security_adjustment.sql:2]: Migration got removed?

Comment (pdittaro) [reply migrations/sql/V2020.09.03.14.10__etl_permit_security_adjustment.sql:2]: Yep

Comment (pdittaro) [reply migrations/sql/V2020.09.03.14.10__etl_permit_security_adjustment.sql:2]: It hasn't run in Dev/Test/Prod

**Review by pdittaro — APPROVED** [review]

**Pull request merged**